Runtime/Systems/EffectTriggerSystems.cs

```diff
@@ -1,4 +1,4 @@
-﻿using Unity.Burst;
+using Unity.Burst;
```

Comment:
Github editor showed some weird invisible character here that might have not been intended. Maybe it's a windows thing, not sure.